### PR TITLE
Update Ameli's konnector to match the last website version

### PR DESCRIPTION
**Pull request by Gara64**
This should fix the current Ameli's konnector state. I've successfully tested  with my account, but as usual, more tests are welcomed :)

### CODE REVIEW

File: server/konnectors/ameli.coffee [hunk 1]
```diff
@@ -98,56 +102,55 @@ parsePage = (requiredFields, healthBills, data, next) ->
...
-        bill =
-            amount: amount
-            type: 'health'
-            subtype: subtype
-            date: moment date, 'DD/MM/YYYY'
-            vendor: 'Ameli'
-            detailsUrl: detailsUrl
+    # request the bill's url
+    request billOptions, (err, res, body) ->
+        if err then next err
```
Comment (nicofrand) [inline server/konnectors/ameli.coffee]: Please, can you log the error and return a locale key instead ?

```coffeescript
log.error err //where log is an instance of the logger of course
return next 'request error'
```

File: server/konnectors/ameli.coffee [hunk 2]
```diff
@@ -80,9 +81,12 @@ logIn = (requiredFields, billInfos, data, next) ->
...
                     strictSSL: false
+                    headers:
+                        'Cookie': res.headers['set-cookie']
+                        'Referer': refererUrl
                     url: reimbursementUrl
 
-                # Last request to get the reimbursements
+                # Last request to get the reimbursements page
                 request reimbursementOptions, (err, res, body) ->
                     if err then next err
```
Comment (nicofrand) [inline server/konnectors/ameli.coffee]: Please, can you log the error and return a locale key instead ?

```coffeescript
log.error err //where log is an instance of the logger of course
return next 'request error'
```

File: server/konnectors/ameli.coffee [hunk 3]
```diff
@@ -98,56 +102,55 @@ parsePage = (requiredFields, healthBills, data, next) ->
...
-    options =
-        method: 'GET'
-        jar: true
-        strictSSL: false
-        url: detailsUrl
+                    healthBills.fetched.push bill if bill.amount?
 
-    # Request to get the pdf url
-    request options, (err, res, body) ->
-        if err? or res.statusCode isnt 200
```
Comment (nicofrand) [inline server/konnectors/ameli.coffee]: Please, can you log the error and return a locale key instead ?

```coffeescript
log.error err //where log is an instance of the logger of course
return next 'request error'
```

**Comment by nicofrand** [comment]
Thanks for the fix !!

**Comment by Gara64** [comment]
@nicofrand Done!

**Comment by nicofrand** [comment]
Thanks, I'll try to test it this evening too and then merge it.

**Comment by nicofrand** [comment]
I can't build it and therefore can't test it:

```
Build entry point...

  rendered build/client/index.js

Entry point built.
Build locales...
/home/nicofrand/www/cozy/konnectors/client/app/locales/fr.coffee:16:6: error: reserved word 'import'
    'import already running': "L'import est déjà en cours."

```
I don't know why, I don't have the issue with the master branch.
Sorry for the delay. Would you mind trying `npm run build` to check if you have the issue too ?

**Comment by nicofrand** [comment]
Sorry, but it did not seem to work. There was nothing in Files and in DataBrowser I have 1 entry with this value as pdfurl : http://localhost:12223/bill.pdf.

**Comment by nicofrand** [comment]
I finally managed to make it work !
One small thing: the konnector seems to fetch the monthly resume PDF, not every reimbursement. The description should specify this, what do you think @Gara64 ?

**Comment by Gara64** [comment]
Sorry for the delay, we're having a team week-end at Cozy :)
Actually, this is not good, I've realized that it only considers one reimbursement per month, no matter how many they are. 
So I'll fix this to deal with all reimbursements in the same month and download each pdf.

**Comment by nicofrand** [comment]
I thought that was considering all reimbursements for a given month since it downloaded the summary of all reimbursements… It just depends whether you prefer one file per reimbursement or one per month… If per month, the description should be more accurate, that's all.

Thanks for the good work and have a great we with the cozy team !

**Comment by Gara64** [comment]
@nicofrand I just pushed a commit that allows to fetch all the reimbursements and not just a monthly summary.
Took me some time, but the week-end was quite... intense :)

**Comment by nicofrand** [comment]
Thanks a lot @Gara64 !

**Pull request merged**